candidate-standard/abstract_tests/collections/ATS_rc-resolutiony-definition.adoc

```diff
@@ -0,0 +1,19 @@
+[[ats_collections_rc-resolutiony-definition]]
+[width="90%",cols="2,6a"]
+|===
+^|*Abstract Test {counter:ats-id}* |*/ats/collections/rc-resolutiony-definition*
+^|Test Purpose |Validate that the resolutiony query parameters are constructed correctly.
```

Comment:
Same here -- maybe a search of the document for all our query parameters to make sure formatting is correct is in order, or was this on purposes?

Reply:
Good idea. Now cracked the back-tick/grave accent `.
@m-burgoyne Over to you.